src/workers/event.worker.ts

```
@@ -4,6 +4,8 @@ import { processOutboxEvent } from '@/shared/events/tasks/process-outbox-event';
 import { TASK_NAMES } from '@/shared/queue/queue.config';
 import { runWorker } from '@/shared/queue/worker-runner';
 import { processOnboardingWebhook } from '@/workers/tasks/process-onboarding-webhook';
+import { processMeteredUsage } from '@/workers/tasks/process-metered-usage';
```

ESLint

[error] `@/workers/tasks/process-metered-usage` import should occur before import of `@/workers/tasks/process-onboarding-webhook`

(import/order)